README.md

```diff
@@ -1,29 +1,89 @@
+
+If you would like to provide feedback, please raise a [GitHub issue](https://github.com/theiagen/public_health_vioinformatics/issues/new).
 
 ### Contributing to the PHB workflows
-Contributions to the workflows contained in this repository are warmly welcomed. Our style guide may be found [here](https://theiagen.notion.site/Style-Guide-WDL-Workflow-Development-bb456f34322d4f4db699d4029050481c) for convenience of formatting.
 
-### Citation
+We warmly welcome contributions to this repository! Our style guide may be found [here](https://theiagen.notion.site/Style-Guide-WDL-Workflow-Development-bb456f34322d4f4db699d4029050481c) for convenience of formatting.
+
+You can expect a careful review of every PR and feedback as needed before merging, just like we do for PRs submitted by the Theiagen team. Our PR template can help prepare you for the review process. As always, reach out with any questions! We love recieving feedback and contributions from the community. When your PR is merged, we'll add your name to the contributors list below!
+
+## Authorship & Responsibility
+
+### Authorship
+
+(Ordered by contribution [# of lines changed] as of 2024-03-28)
+
+* **Sage Wright** (@sage-wright) - Conceptualization, Software, Validation, Supervision
+* **Inês Mendes** (@cimendes) - Software, Validation
+* **Curtis Kapsak** (@kapsakcj) - Conceptualization, Software, Validation
+* **Frank Ambrosio** (@frankambrosio3) - Conceptualization, Software, Validation
+* **James Otieno** (@jrotieno) - Software, Validation
+* **Michelle Scribner** (@michellescribner) - Software, Validation
+* **Kevin Libuit** (@kevinlibuit) - Conceptualization, Project Administration, Software, Validation, Supervision
+* **Robert Petit** (@rpetit3) - Software, Validation
+* **Emma Doughty** (@emmadoughty) - Software, Validation
+* **Andrew Page** (@andrewjpage) - Project Administration, Software, Supervision
+* **Kelsey Kropp** (@kelseykropp) - Validation
+* **Emily Smith** (@emily-smith1) - Validation
+* **Joel Sevinsky** (@sevinsky) - Conceptualization, Project Administration, Supervision
+
+### External Contributors
+
+We would like to gratefully acknowledge the following individuals from the public health community for their contributions to the PHB repository:
+
+* **Ash O'Farrel** (@aofarrel)
+* **Holly Halstead** (@HNHalstead)
+* **Sam Baird** (@sam-baird)
+
+### Maintainence Commitment
```

Comment:
Check out the updated [PHA4GE Pipeline Best Practice](https://github.com/pha4ge/public-health-pipeline-best-practices/blob/main/docs/pipeline-best-practices.md#10c-pipeline-maintenance-statement) for this section. 

To adhere to PHA4GE practices, I'd recommend we re-title this section to something more like`#Maintaining PHB Pipelines` and, in addition to our commitment, indicate that workflows were written using a standard workflow language (WDL), docker images based on StaPH-B docker builds, and version controlled with detailed technical notes to lower the barrier of pipeline upkeep from the public health community at large. These types of details seem to be more in line with the PHA4GE Pipeline Best Practices